images/docker-compose-mca.yml

```diff
@@ -8,6 +8,6 @@ services:
       labels:
         org.opencontainers.image.source: https://github.com/cnpem/epics-in-docker
       args:
-        REPONAME: mca
+        APP_DIRS: /opt/epics/modules/mca
```

Comment:
Also mention in commit message that REPONAME wasn't relevant for these images, and that they were moved to use `APP_DIRS` so as not to break them.